### DISCUSSION TITLE
Universe

### DISCUSSION THREAD

**Discussion by OlliSant** [a-cosmology-group/acg#24] [Metaphysics and Philosophy]
To join the discussion in this new format: 

The Universe, the whole universe, not the observable universe is eternal and boundless immensely big MultiBang universe. The BB is about the local universe, those galaxies which are from the same beginning as Milky Way and most of the nearby galaxies.

**Comment by RedshiftDrift** [comment]
If the cosmological redshift is interpreted as a *velocity* (an increasing distance with time), then every galaxy we have observed until now shows this velocity.  Therefore what you call "local universe" is everything we see.

Why do you assume "multi-Bangs" when we see only one?

**Reply by OlliSant** [reply]
There is no expansion, it is there only because of the theory, not because of observations, redshift or others, or facts. When we change the theory of time and space and matter (GR and BB) and interpretations of the observations, there is no expansion. And the theory of the universe must be of the whole universe, not of the observable universe, which is smaller. When we abandon the BB- model, we will make new inventions and the science of cosmology will be free and flourishing as newer before! We explore the galaxies in a different way.

Cosmology is always philosophy and astronomy together, and philosophy is the main thing. It was wrong to start to study cosmology independently of metaphysics, philosophy, as happened about 1960. Because philosophy is much better when we handle totalities, especially this unknown totality, the universe.

**Comment by OlliSant** [comment]
There are many bangs, and smokes after them, when we start to search them. And one bang, one beginning, is impossible, which should have been understood immediately in 1920. Einstein was wrong to see BB as good at all.

Well, give us your real thoughts, please!

**Reply by khuramonline** [reply]
"Bang" is like a thinking pattern. Now I get to know that this pattern can also work for a non-expansionist mind.

**Reply by OlliSant** [reply]
Yes, there has been many Bangs and crushes and there are many clouds after them. Why not found? There must not be them, says BB- theory,  so astronomy does not search them.

**Reply by khuramonline** [reply]
> Yes, there has been many Bangs and crushes and there are many clouds after them. Why not found? There must not be them, says BB- theory, so astronomy does not search them.

This is not answer to my point. You should have explained how do you, being non-expansionist, reconcile with bang or bangs.

**Reply by OlliSant** [reply]
What it is, reconcile with bang or bangs? I don't have to prove non- expansion, I suppose it, and then try to prove that such theory is better. That is the proof, theory will be found better eventually.

**Comment by OlliSant** [comment]
My book where are the articles too of this matter http://www.santavuori.com/Filosofia2019.html

**Comment by OlliSant** [comment]
When we look all the information in astronomy, which we now have, we can clearly say which kind of universe we have in the sky and everywhere. We do see already far enough. We just must forget our previous thinking and mathematics and theories. We have the deep sky photos of Hubble and Webb, the maps of the galaxies, and animations. That is all we need. What is the steady state of the universe, we can see of that fact: 

what is same far away and same near us?

 That is the web of the galaxies. How this web develops from initial smokes, we can see from animations. 

So it needs only little scientific thinking, and it is clear. We know already the physics of the universe. The universe is galaxies, that is the static thing, and the development from clouds is its dynamic. The beginning and development is **local** thing, the universe of galaxies is eternal thing everywhere. For the whole universe there is no beginning, no expansion. And even for the local (part)universe there is no expansion, only in the beginning of the explosion or whatever it is its beginning. Smoke anyway, every celestial body is from some explosion and smoke after that.

You must think this over and answer this. This is important, if true.
